integration_tests/src/main/python/cache_test.py

```diff
@@ -20,22 +20,24 @@
 from spark_session import with_cpu_session, with_gpu_session
 from join_test import create_df
 from marks import incompat, allow_non_gpu, ignore_order
-from parquet_write_test import parquet_decimal_struct_gen
 
-enableVectorizedConf = [{"spark.sql.inMemoryColumnarStorage.enableVectorizedReader" : "true"},
-                        {"spark.sql.inMemoryColumnarStorage.enableVectorizedReader" : "false"}]
+conf = [{"spark.sql.inMemoryColumnarStorage.enableVectorizedReader": "true",
```

Comment:
I have addressed your concern I think

Reply:
My understanding you addressed `enableVectorizedConf` but not granular parameters. It's minor but to avoid some repetitiveness Instead of whole lengthy conf parameters we could have parameters to the tune :

```python
@pytest.mark.parametrize('enable_vectorized_conf', ['VECTORIZED_ON', 'VECTORIZED_OFF'], ...) 
def test_passing_gpuExpr_as_Expr(enable_vectorized_conf):
   assert_gpu_and_cpu_are_equal_collect(
      ...,
      conf = { 'spark.sql.inMemoryColumnarStorage.enableVectorizedReader': enable_vectorized_conf == 'VECTORIZED_ON' }
   )
```